lib/debuglet.js

```diff
@@ -229,9 +227,11 @@ Debuglet.prototype.scheduleBreakpointFetch_ = function(seconds) {
  */
 Debuglet.prototype.updateActiveBreakpoints_ = function(breakpoints) {
   var that = this;
-
   var updatedBreakpointMap = this.convertBreakpointListToMap_(breakpoints);
-  that.logger_.breakpoints(Logger.INFO, 'Server breakpoints:', updatedBreakpointMap);
+
+  if (breakpoints.length) {
+    that.logger_.breakpoints(Logger.INFO, 'Server breakpoints:', updatedBreakpointMap);
```

Comment:
At the INFO level, wouldn't it still be informative to know that there are no breakpoints on the server?